### ISSUE TITLE
Don't hide the Problem pane every now and then

### ISSUE THREAD

**Issue by ZzZombo** [Microsoft/vscode#54781]
Issue Type: <b>Bug</b>

It just keeps hiding itself every so othen, even if I have active issues in my files. Makes me irritated by having to reopen it each time it does so.

VS Code version: Code 1.25.1 (1dfc5e557209371715f655691b1235b6b26a06be, 2018-07-11T15:40:20.190Z)
OS version: Linux x64 4.15.0-24-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Pentium(R) CPU  N3540  @ 2.16GHz (4 x 2665)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: unavailable_software<br>video_encode: enabled<br>vpx_decode: unavailable_software<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|7.68GB (0.13GB free)|
|Process Argv|/usr/share/code/code --unity-launch|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (67)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|9.0.0
project-manager|ale|8.0.0
vscode-color|ans|0.4.5
color-info|bie|0.5.0
npm-intellisense|chr|1.3.0
path-intellisense|chr|1.4.2
regex|chr|0.2.0
vscode-css-modules|cli|0.2.3
gitignore|cod|0.5.0
bracket-pair-colorizer|Coe|1.0.59
vscode-eslint|dba|1.4.12
vscode-faker|dee|1.3.0
vscode-npm-source|dku|1.0.0
git-extension-pack|don|0.1.3
githistory|don|0.4.2
jupyter|don|1.1.4
xml|Dot|2.3.1
gitlens|eam|8.5.2
EditorConfig|Edi|0.12.4
vscode-npm-script|eg2|0.3.5
json-tools|eri|1.0.2
auto-rename-tag|for|0.0.15
vscode-mysql|for|0.3.0
gc-excelviewer|Gra|2.1.24
beautify|Hoo|1.3.2
icon-fonts|idl|2.1.0
search-node-modules|jas|1.3.0
c-cpp-flylint|jbe|1.1.1
svg|joc|0.1.1
Lua|key|0.0.9
vscode-github|Kni|0.30.0
graphql-for-vscode|kum|1.12.0
git-indicators|lam|2.1.1
node-module-intellisense|lei|1.5.0
vue|liu|0.1.5
sftp|lix|1.2.5
ecdc|mit|0.12.0
HTMLHint|mka|0.5.0
vscode-filesize|mkx|2.1.1
perl-debug|mor|0.4.1
hg|mrc|1.2.3
vscode-language-pack-ru|MS-|1.25.2
cpptools|ms-|0.17.7
csharp|ms-|1.15.2
typescript-javascript-grammar|ms-|0.0.42
vscode-extension-auto-import|Nuc|1.4.3
indent-rainbow|ode|7.2.2
vscode-versionlens|pfl|0.21.1
vscode-css-peek|pra|2.1.0
quicktype|qui|12.0.5
vscode-sort-json|ric|1.13.0
vscode-icons|rob|7.24.0
git-merger|sha|0.4.1
html5-boilerplate|sid|1.0.3
vscode-fileutils|sle|2.10.3
code-spell-checker|str|1.6.10
bootstrap4-vscode|the|4.3.0
pdf|tom|0.3.0
gitflow|vec|1.2.1
gitblame|wad|2.4.2
nodejs-extension-pack|wad|0.1.9
vscode-todo-highlight|way|0.5.12
vscode-import-cost|wix|2.7.2
change-case|wma|1.0.0
JavaScriptSnippets|xab|1.7.0
html-css-class-completion|Zig|1.17.1
vscode-open-in-github|ziy|1.3.5

</details>

**Comment by sandy081** [comment]
@ZzZombo I doubt that this might be done by an extension you installed. Can you please reproduce it by disabling all extensions?

**Comment by ZzZombo** [comment]
Oh, sorry, somehow I didn't get any prior notification of activity here. I would test w/o extensions, but I'm at loss how to deactivate ALL of them, more than 30; not going to do it manually, you know. But I hope this will help, 1) didn't happen on Windows with the almost exact same setup (it occurred first on Linux), and 2) stopped to happen after I moved the Problem pane before the built-in Terminal. Then I moved it back to the original location and it no longer happens as well.30.07.2018, 22:01, "vscodebot[bot]" <notifications@github.com>:Closed #54781.
